pysand/fluidproperties.py

```diff
@@ -1,7 +1,23 @@
 import numpy as np
+import logging
+import pysand.exceptions as exc
 
+logger = logging.getLogger(__name__)
 # Models from DNVGL RP-O501, equation references in parenthesis
 
+def validate_fluid_props(**kwargs):
+    """
+        Validation of all input parameters that go into erosion models;
```

Comment:
should be input parameters that go into fluid properties models

Reply:
My bad...